static/docs/user-guide/glossary.md

```diff
@@ -0,0 +1,30 @@
+# Glossary
+
+This guide is aimed to familiarize the users with definitions to relevant DVC
+concepts and terminologies which are frequently used.
+
+## Workspace directory
+
+Also abbreviated as workspace, it is the root directory of a project where DVC
+is initialized by running `dvc init` command. Therefore, this directory will
+contain a `.dvc` directory as well.
+
+## Cache directory
+
+DVC cache is a hidden storage which is found at `.dvc/cache`. This storage is
+used to manage different versions of files which are under DVC control. For more
+information on cache, please refer to the this
+[guide](/doc/commands-reference/config#cache).
+
+## Pipeline
+
+In our documentation, it has also been referred to as a computation graph. It is
+an ordered set of some executable commands, each of them stored in a Dvcfile. To
+get familiar with its usage, please refer to this
+[tutorial](/doc/get-started/example-pipeline#example-pipelines).
+
+## DAG
```

Comment:
it's the same as previous?

Reply:
@shcheklein it seemed like similar when I read about DAG from some resources online.

Reply:
Definitely requires disambiguation. TBHI think  this glossary is not a simple task, please take your time trying to unwind these concepts from existing docs before suggesting how to explain them and where to link from the glossary (:

Reply:
@algomaster99 did you have a chance to check the suggested groups of terms for the glossary I listed in the description of #424 ? In there "Pipeline" and "DAG" are actually separate but related entries.